src/modm/platform/clock/stm32/rcc.hpp.in

```diff
@@ -368,6 +376,7 @@ public:
  %% else
 		const uint8_t pllP;
  %% endif
+ 		const uint8_t pllQ = 15;
```

Comment:
Why default to 15?

Reply:
As far as I remember there was some reason not to set default values here.

Reply:
I've changed it so it only gets written and enabled if it is actually set.